include/cocaine/api/storage.hpp

```diff
@@ -97,13 +86,13 @@ storage_t::get(const std::string& collection, const std::string& key) {
     try {
         msgpack::unpack(&unpacked, blob.data(), blob.size());
     } catch(const msgpack::unpack_error& e) {
-        throw storage_error_t("object is corrupted");
+        throw std::system_error(std::make_error_code(std::errc::invalid_argument));
```

Comment:
If the main reason is to save some code bytes, then I think that there is no need to indroduce another entity. If there are some other reasons, please explain.

Internally `std::system_error` has a constructor overload which accepts `std::error_code`, which can be constructed implicitly using error code enumeration if it implements `std::is_error_code` trait. However the `std::errc` implements only `std::is_error_condition` trait for some reasons.